src/wp-includes/block-supports/layout.php

```diff
@@ -170,8 +204,11 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 		$used_layout = $default_layout;
 	}
 
-	$class_name = wp_unique_id( 'wp-container-' );
-	$gap_value  = _wp_array_get( $block, array( 'attrs', 'style', 'spacing', 'blockGap' ) );
+	$container_class = wp_unique_id( 'wp-container-' );
```

Comment:
Can this be in the function so it is `wp_get_block_class_names` or somethign?

Reply:
I think it is better to keep it separate for now as this unique container id is different to the utility class names being generated by the new temporary method, and may or may not remain after that method is superseded by the style engine functionality.

Reply:
Do you mean `wp_get_layout_classes()` is intended to be temporary? 

I'd rather avoid adding it if it's know that it will be deprecated. It avoids the tax of having to maintain backward compatibility.

Reply:
Good point @peterwilsoncc, I was overlooking the backwards compat issues of global public `wp_` methods. I will take another look at this from that perspective.

Reply:
@peterwilsoncc  - given uncertain future of that method I have pulled the code back into the main method, and opened up a [PR to do the same in Gutenberg](https://github.com/WordPress/gutenberg/pull/41885)